content/ux-strat-bootcamp/week-one/thursday/data-dashboard-analysis/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Data Dashboard Analysis
+content_type: project
+ready: true
+---
+
+A data dashboard is an information management tool that visually tracks, analyses and displays key performance indicators (KPI), metrics and key data points to monitor the health of a business, department or specific process.
+
+A wireframe is a simple visual guide that represents the skeletal framework of a digital product. 
+
+**Task**
+Umuzi gets daily inquiries from applicants which are managed by our internal Community Management team. Analyse the dashboard figure Umuzi Recruit Application Inquiries (in the next slide) and improve it. Prepare a 10min presentation to showcase your dashboard.
+ 
+Think through which information you would like to track and communicate. Consider the following aspects:
+Who is the user of the dashboard?
```

Comment:
If you look at the site that gets rendered, all this text will probably show up on one line. 

Use dashes to make lists of points. Eg something like this:

Think through which information you would like to track and communicate. Consider the following aspects:

- Who is the user of the dashboard?
- Why would the user need to use the dashboard?

Do your own additional research on relevant information to include in an applicant inquiries dashboard.